lib/alerts_viewer_web/live/bus_live.ex

```diff
@@ -150,7 +157,29 @@ defmodule AlertsViewerWeb.BusLive do
   def seconds_to_minutes(nil), do: nil
 
   def seconds_to_minutes(seconds) do
-    (seconds / 60) |> Float.round(1)
+    (seconds / 60) |> round
+  end
+
+  def display_minutes(assigns) do
+    list_of_minutes =
+      assigns.stats_by_route
+      |> RouteStats.vehicles_instantaneous_headway_secs(assigns.route)
+      |> Enum.sort(:desc)
+      |> Enum.map(&seconds_to_minutes/1)
+
+    assigns = assign(assigns, :list_of_minutes, list_of_minutes)
+
+    ~H"""
+    <%= for m <- @list_of_minutes do %>
+      <span class="after:content-[','] last:after:content-['']">
+        <%= if m >= 20.0 do %>
```

Comment:
suggestion (non-blocking): This works fine, but another option might be a conditional font-weight class on the parent `span` element. The one benefit of that I can think of is that you wouldn't have to repeat the `<%= m %>` bit.